man/kernel-command-line.xml

```diff
@@ -322,6 +322,15 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>systemd.default_timeout_start_sec=</varname></term>
+
+        <listitem>
+          <para>Overwrites the default start job timeout <varname>DefaultTimeoutStartSec</varname> at boot. For details,
```

Comment:
one more fix here: please suffix config options with "=", i.e. make this `DefaultTimeoutStartSec=`, this is how we usually write such settings, to clarify that they are assignment options.